.github/workflows/ci.yaml

```
@@ -91,3 +91,50 @@ jobs:
           retention-days: 1
           name: image-tar-frrk8s
           path: /tmp/frrk8s.tar
+  e2e:
+    runs-on: ubuntu-22.04
+    needs:
+      - unit-tests
+      - build-test-images
+      - commitlint
+    strategy:
+      fail-fast: false
+      matrix:
+        ip-family: [ipv4, ipv6, dual]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install kernel modules
+        run: |
+          sudo apt-get update
+          sudo apt-get install linux-modules-extra-$(uname -r)
+
+      - name: Download frr8ks images
+        uses: actions/download-artifact@v3
+        with:
+          path: image
+
+      - name: Load image
+        working-directory: image
+        run: |
+          docker load -i image-tar-frrk8s/frrk8s.tar
+
+      - name: E2E
+        run: |
+          IP_FAMILY="${{ matrix.ip-family }}"
+          IP_FAMILY=$IP_FAMILY make kind-cluster
+          SKIP="none"
+          if [ "${{ matrix.ip-family }}" == "ipv4" ]; then SKIP="$SKIP|IPV6|DUALSTACK"; fi
+          if [ "${{ matrix.ip-family }}" == "dual" ]; then SKIP="$SKIP|IPV6"; fi
+          if [ "${{ matrix.ip-family }}" == "ipv6" ]; then SKIP="$SKIP|IPV4|DUALSTACK"; fi
+
+          IMG=quay.io/metallb/frrk8s:dev-amd64 make load-on-kind
+          IMG=quay.io/metallb/frrk8s:dev-amd64 make deploy
+          GINKGO_ARGS="--skip $SKIP" make e2etests
```

misspelled the command
```suggestion
          GINKGO_ARGS="--skip $SKIP" make e2etest
```

---

ci passed with `make: Nothing to be done for 'e2etests'.`